src/Widgets/Banner.vala

```diff
@@ -22,12 +22,12 @@ const string BANNER_STYLE_CSS = """
     @define-color banner_bg_color %s;
     @define-color banner_fg_color %s;
 
-    .banner { 
+    .banner {
         transition: all %ums ease-in-out;
     }
 """;
 
-const string DEFAULT_BANNER_COLOR_PRIMARY = "#454241";
+const string DEFAULT_BANNER_COLOR_PRIMARY = "#68758e";
```

Comment:
We'll want to revert this change. The old color is Pop brown, this new color is the elementary slate.